src/external/stb_truetype.h

```diff
@@ -500,7 +500,11 @@ int main(int arg, char **argv)
 #define __STB_INCLUDE_STB_TRUETYPE_H__
 
 #ifdef STBTT_STATIC
+#ifdef __GNUC__
+#define STBTT_DEF static inline
```

Comment:
Why?

In any case, external libraries shouldn't be modified in raylib but in the original repositories.

Reply:
GCC / MinGWx64 would give a warning without the inline being there. Here is more info about INLINE on the official GCC website.

https://gcc.gnu.org/onlinedocs/gcc/Inline.html

Reply:
I'm afraid that link does not answer my question, I know what is an inline function.

Reply:
In GCC / MinGWx64 you will get these WARNINGS : defined but not used [-Wunused-function]

To remove those warnings without supressing it, you use the keyword INLINE.